staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -130,6 +130,11 @@ type KubeletConfiguration struct {
 	// Default: 10250
 	// +optional
 	Port int32 `json:"port,omitempty"`
+	// readOnlyBindAddress is the address for the Kubelet to serve on with
+	// no authentication/authorization. Defaults to the secure serving address if not set.
```

Comment:
nit: "secure serving address" may not be obvious for a reader. Perhaps just call it "address" or "the value of Address field"